seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -192,9 +192,9 @@
     },
     {
       "id": "587d8255367417b2b2512c75",
-      "title": "How to Create a Circular Queue",
+      "title": "Create a Circular Queue",
       "description": [
-        "In this challenge you will be creating a Circular Queue. A circular queue is basically a queue that writes to the end of a collection then begins over writing itself at the beginning of the collection. This is type of data structure has some useful applications in certain situations. For example, a circular buffer can be used in streaming media. Once the queue is full, new media data simply begins to overwrite old data.",
+        "In this challenge you will be creating a Circular Queue. A circular queue is basically a queue that writes to the end of a collection then begins over writing itself at the beginning of the collection. This is type of data structure has some useful applications in certain situations. For example, a circular queue can be used to stream media. Once the queue is full, new media data simply begins to overwrite old data.",
```

Comment:
If I'm not confusing this for something else, the circular queue technically isn't responsible for the *streaming of media* as the current sentence indicates - it's useful when storing the streaming media.

I'd suggest the following change
```diff
-to stream media
+for streaming media
```